common/src/main/proto/TransportMessages.proto

```diff
@@ -173,6 +175,14 @@ message PbRegisterShuffleResponse {
   repeated PbPartitionLocation partitionLocations = 2;
 }
 
+message PbHeartbeatFromClient {
+  repeated int32 shuffleId = 1;
+}
+
+message PbHeartbeatFromClientResponse {
+  repeated int32 unknownShuffleId = 1;
```

Comment:
unknownShuffleIds?

Reply:
I prefer to use singular nouns for repeated fields, because the generated class's method names are like:
```
getUnknownShuffleIdList
getUnknownShuffleIdCount
getUnknownShuffleId(int index)
```
It will be wired to use plural nouns

Reply:
I agree to use singular too